pkg/build_tar.py

```diff
@@ -23,7 +23,7 @@
 from rules_pkg import archive
 from absl import flags
 
-from helpers import GetFlagValue
+from helpers import GetFlagValue, UnquoteAndSplit
```

Comment:
I moved this function out of `build_tar.py` to `helpers.py` so it could be used by `build_zip.py` as well. I renamed it to match the other function in that library, let me know if that doesn't make sense.

Reply:
That makes sense.
Could you do this in a distinct refactoring PR, so we can start adding unit tests for it.

It should also have a meaningful name. unquote_and_split its not what it really does. That implies we unquote args.  It is simply splitting on the separator and allowing \= in the string. It is also a mess of unexpected semantics. It looks like   na\xffme=value would result in 'nax25me' value, instead of the expected 'na%me', 'value' or the (also correct) 'na\xffme', 'value'

How about   SplitNameValuePairAtSeparator(arg, sep='=')

Reply:
Thanks for pulling that out.

Reply:
NP! I'm going to rebase this and incorporate the feedback soon (lots of great feedback, thanks)